.github/workflows/ci.yml

```diff
@@ -71,52 +71,45 @@ jobs:
         linux-local-args: '["--toolkit", "--override"]' 
       if: ${{ matrix.enable_cuda }}
 
-    - name: Install Miniconda
```

Comment:
Could this PR be limited to adding the GPU runner rather than changing entire CI?

Reply:
Sure, I was just trying to save some CI time and make how I did the GPU runner match the rest of the CI, I will make as small of a change as I can

Reply:
Thanks for trying to bring innovations. Showcase them with the GPU CI and we will consider updating the rest CI.